applications/dashboard/models/class.logmodel.php

```diff
@@ -394,7 +394,7 @@ public function getIDs($iDs) {
      * @param bool $limit The database limit.
      * @return array Returns a data set.
      */
-    public function getWhere($where = false, $orderFields = '', $orderDirection = 'asc', $offset = false, $limit = false) {
+    public function getWhere($where = false, $orderFields = '', $orderDirection = 'asc', $offset = false, $limit = false, $orWhere = false) {
```

Comment:
I would like to get away from using `false` as the default for these types of things. I know it is consistent with old code, but a boolean `false` has a very specific meaning. I think `null` is a more appropriate default. It also has a specific meaning: nothing. I think that's what we're trying to say with the default value of this parameter: we don't have any or-where conditions. Nullable parameters will also play nicely with type declarations.